### DISCUSSION TITLE
Config for showing Dial-In numbers on the frontend.

### DISCUSSION THREAD

**Discussion by 1ay1** [igniterealtime/openfire-pade-plugin#309] [Q&A] [answered]
Reading this guide on how to get dial in numbers and PIN shown on the frontend.
https://community.jitsi.org/t/working-configuration-for-jitsi-jigasi-with-asterisk-sip/53039

How can we add the following two keys to the config?
1) "dialInNumbersUrl"
2) "dialInConfCodeUrl"

**Comment by deleolajide** [comment]
Can you please stop asking questions in issues section instead of the discussion section.

**Comment by deleolajide** [comment, marked as answer]
> How can we add the following two keys to the config?

Edit this file https://github.com/igniterealtime/openfire-pade-plugin/blob/9bd1f0c0f808c368a6f4dcb75d93efdfa467a7e8/src/webapp/config_custom.js#L1

Add 

config.dialInNumbersUrl ="xxxxxxxxxxxx";
config.dialInConfCodeUrl = "zzzzzzzzzzzzzzzz";

If you want to make it permanant, then clone Pade and make the changes to your clone and rebuild repo or unzip pade.jar, edit file and repackage
